tests/usm/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+file(GLOB test_cases_list *.cpp)
```

Comment:
It's generally bad CMake practice to glob for source files, it's better to list them explicitly in order to be able to properly track file dependencies.

Reply:
This decision implemented across all repository. Fixing this task out of scope this pull request

Reply:
We could start doing it properly just here for now, but I'm happy either way